redi-connect-backend/server/datasources.json

```diff
@@ -5,7 +5,7 @@
   },
   "mongodb": {
     "host": "",
-    "port": 0,
+    "port": 27018,
```

Comment:
@dyyyl this would cause issues for people running the default `mongod --dbpath .` command without the additional `--port 27018`.

I see two options:
1. revert this change back to 0
2. run `mongod` by default on port 27018

I prefer option #2, to mitigate the possibility of conflict with other running mongodb servers on developers' boxes. What do you think about creating a sort of `./start-db.sh` script in the root folder? Or even better, one single `start-dev.sh` command in the root folder to start all components (db, admin, loopback, front)?